client/my-sites/plans/main.jsx

```diff
@@ -36,7 +36,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
oh! I didn't realize that we have removed the ".isJetpack" check in the plans component ... is this a good idea? I mean, it sounds like it would be too easy to forget to pass this param and break something. I would argue that it would be a good idea to restore the "isJetpack" check in `plans-features-main`, and just add some logic there to cover the AT special case. Until know, anybody could create a new plans page without a lot of digging, and the type of plans would be automatically selected depending on the type of the site. This change makes that the type-of-plans selection logic needs to be added in each component or controller that use an instance of `plans/main`.
If we don't want to do that, at least I would remove this default value here, so each instance of plans would need to explicitly declare its value.